translations/ui-data-import/en_US.json

```diff
@@ -900,5 +900,7 @@
     "modal.cancelRunningJob.message": "Are you sure that you want <b>to cancel this import job?</b><p><b>Note:</b> Cancelled jobs cannot be restarted. Records created or updated before{break}the job is cancelled cannot yet be reverted.</p>",
     "modal.cancelRunningJob.cancel": "No, do not cancel import",
     "modal.cancelRunningJob.confirm": "Yes, cancel import job",
+    "modal.confirmJobFinished.header": "Import job finished before cancellation",
```

Comment:
Please check en.json for the same translations. I'm not sure if the rest of the files will be populated based on en_US.json